### DISCUSSION TITLE
USB shield for Pro mini 3.3v vs USB shifter which required 5v?

### DISCUSSION THREAD

**Discussion by macosrom** [azzajess/USB-Shifter-to-Thrustmaster-Wheelbase#11] [Q&A]
Hi,

Recently, I were following this project, and now stuck with no power on USB Shifter situation.

USB Shifter with USB A connection with normal computer is working fine. but to monitor each gear output of USB Shifter, I connect it to USB shield for Pro mini along with Pro mini board (3.3v) and TTL adapter. The USB Shifter has internal LED and it doesn't turns and there isn't monitor out.  

I verify that  Pro mini board (3.3v) and TTL adapter is working fine.

I assumed that maybe USB shield has malfunction or USB Shifter required 5v and USB shield for Pro mini along with Pro mini board (3.3v) can't provide 5v

Is it can happen due to insufficient of voltage? or something have to be done?

**Comment by azzajess** [comment]
Whoops posted before in wrong account


Hmmmm
Which type of shifter is it? The eBay/AliExpress ones?
What's the wheel base?
If you take the ttl adaptor out of the circuit so it's just the Arduino and USB shield connected between the wheel and shifter, do you get anything?

If I remember correctly these USB shields don't like 5v unless you make a modification to something. Sites down so can't double. Also I don't fully understand it but USB protocol works better at 3v (again sites down so can't fact check myself)

But insufficient voltage shouldn't be an issue as long as 3v is actually coming from the wheelbase. But maybe shifter is current hungry?

**Reply by macosrom** [reply]
Sorry for late response and thanks for help.

I was using Thrustmaster T300 Base and shifter were from aliexpress.
https://www.aliexpress.com/item/1005004132471839.html?gatewayAdapt=4itemAdapt

I was trying to read value from USB output of shifter to follow usb shield mod so didn't try to connect it to base.
And I confirmed that usb shield were functional with other usb device so it seems like above shifter were requires 5v when it connected to USB mode.

And Recently, I migrate my sim rig from Thrustmaster to Fanatec and utilize this shifter as sequential shifter with hard wiring.

All thing can happen from your project sharing, I appreciated.
